spinalcordtoolbox/registration/algorithms.py

```diff
@@ -329,6 +337,220 @@ def register_step_label(src, dest, step, verbose=1):
     return warp_forward_out, warp_inverse_out
 
 
+def register_step_dl_multimodal_cascaded_reg(src, dest, step, verbose=1):
+    """
+    """
+    # Preprocessing steps - TODO maybe not include these here so there is an option for the user to test different ones
```

Comment:
Note to resolve this TODO

Reply:
@EvanBeal Did you have any thoughts about which options you would want to expose to the user to be able to change? (e.g. changing isotropic resolution, perhaps?)

It's tricky, because the current `-param` options don't give us much flexibility in terms of exposing options to the user. The closest I can think of is like, `shrink`? 

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/2ea3643585099860751ce2bf5fb76919dd580f86/spinalcordtoolbox/scripts/sct_register_multimodal.py#L189-L191

But the usage here is kind of arcane. For example, if we want to use 0.9mm isotropic resolution, we would have to specify a `shrink` value of 1/0.9 == `1.11..`. 

It might be easiest to just... leave the defaults as is, then. :sweat:

Reply:
> Did you have any thoughts about which options you would want to expose to the user to be able to change? (e.g. changing isotropic resolution, perhaps?)

The preprocessing that could be changed by the user is indeed this isotropic resolution of 1 mm. I have set this value by default as I have done most of the tests in my project with this preprocessing but in fact as the registration models are trained on synthetic data, they are not specific to data with an isotropic resolution of 1 mm. Therefore, we could expect good registration results for data with a different preprocessing as well, but there are more uncertainties as it has not been test in depth.

So, I think that the ideal solution would have been to set the isotropic resolution of 1 mm and then have an option to potentially change it (or maybe even remove this preprocessing step). But I agree that it's quite complex because we don't really want to add a param (and need to refactor everything) just for this special case and as you said using the `shrink` parameter may be a bit tricky. 

Therefore, I guess that for now we could either leave it as it stands or remove it and mention that the user can do this preprocessing step before using the method if he doesn't observe satisfactory results. 

I think that the first option of leaving it as it is now is probably the best to facilitate the use of the method.